bootstore/src/messages.rs

```diff
@@ -110,4 +116,13 @@ pub enum NodeError {
 {rack_uuid}, epoch: {epoch}"
     )]
     MissingKeySharePrepare { rack_uuid: Uuid, epoch: i32 },
+
+    #[error("DB error: {0}")]
+    Db(db::Error),
+
+    #[error(
+        "'KeySharePrepare' messages are not allowed for epoch 0.
+Please send an 'Initialize' message"
```

Comment:
Stray newline in this error message also

Reply:
good catch!

Reply:
fixed.